src/tests/JIT/Regression/JitBlue/Runtime_91175/Runtime_91175.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.aa
+
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+using Xunit;
+
+public class TestClass
+{
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    public static Vector256<int> Method0() => Avx2.ShiftRightArithmetic(Vector256<int>.AllBitsSet, Vector128<int>.AllBitsSet);
```

Comment:
Could you update the test using `Sse2.ShiftRightArithmetic`? That way it could exercise Mono runtime as well, since Mono hasn't supported emitting intrinsics for Avx2 and Vector256.

Reply:
Adjusted to have the `Isa.IsSupported` checks and test both cases